app/cli/index.js

```diff
@@ -1,21 +1,22 @@
 // This is a CLI tool, using console is OK
+//
 /* eslint no-console: 0 */
-const {spawn, exec} = require('child_process');
-const {isAbsolute, resolve} = require('path');
-const {existsSync} = require('fs');
-const pify = require('pify');
 const args = require('args');
 const chalk = require('chalk');
 const opn = require('opn');
 const columnify = require('columnify');
 const got = require('got');
 const ora = require('ora');
 const api = require('./api');
-
+const isDev = require('electron-is-dev');
+const promiseFinally = require('promise.prototype.finally');
+promiseFinally.shim();
 const PLUGIN_PREFIX = 'hyper-';
 
-let commandPromise;
-
+const configLocation = process.platform === 'win32' ? process.env.userprofile + '\\.hyper.js' : '~/.hyper.js';
```

Comment:
Will crash with new location of `.hyper.js`

Reply:
@jaydh